src/main/resources/templates/datewarehouse/datawarehouse-config.json

```diff
@@ -249,6 +249,16 @@
 				"name": "effective_ars",
 				"type" : "array<bigint>",
 				"comment" : "The list of access requirement ids that apply to the entity at the time the snapshot was taken."
+			},
+			{
+				"name": "annotations",
+				"type": "string",
```

Comment:
Are there string limit sizes for Athena? I know there's a 16mb string limit size for snowflake string columns. Although, id probably parse these are a "variant" column

Reply:
There is no documented limit, but I'm not worried about Athena since we do not plan to actually read this data through it. We should be under the 16mb since our apis have a max limit of 2mb per request.